tokio-util/src/codec/length_delimited.rs

```diff
@@ -79,26 +79,28 @@
 //! frame head in the yielded `BytesMut`.
```

Comment:
> including the frame head in the yielded `BytesMut`.

Perhaps the comment here should also be updated to `omitting` instead of `including`?

Reply:
Yes, I forgot this. All `including`/`omitting` is corrected now.